Overview.bs

```diff
@@ -419,9 +423,9 @@ instance list</a> as |instance|:
 
   6. Let |animateFunction| be |definition|'s <a>animate function</a>.
 
-  9. Let |timestamp| be a {{DOMHighResTimeStamp}} indicating the current frame start time.
+  9. Let |timestamp| be a DOMHighResTimeStamp indicating the current frame start time.
```

Comment:
I think this should be linkable to https://www.w3.org/TR/hr-time-2/#dom-domhighrestimestamp